src/applications/auth/manifest.json

```diff
@@ -3,6 +3,9 @@
   "entryFile": "./auth-entry.jsx",
   "entryName": "auth",
   "rootUrl": "/auth/login/callback",
-  "noRouting": true,
-  "production": true
```

Comment:
`noRouting` and `production` do nothing, so I removed them since they're kinda misleading.